nuget/Microsoft.Windows.CsWinRT.Authoring.targets

```diff
@@ -253,30 +260,30 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <!-- When packing, include all necessary DLLs and the targets file for DLL copying on the native side -->
     <ItemGroup>
 
-      <Content Include="$(CsWinRTPath)lib\net6.0\WinRT.Host.Shim.dll">
+      <Content Condition="'$(CsWinRTAotExportsEnabled)' != 'true'" Include="$(CsWinRTPath)lib\net6.0\WinRT.Host.Shim.dll">

```

Comment:
You may want to put the condition on the ItemGroup instead of duplicating them everywhere.

Reply:
This is not added to all items in this group. For instance, the .winmd etc. are always copied. I suppose I could split those to be in a separate group though, to keep things simpler, so we'd have a group for "always copied" and one for "only copied when not publishing with NAOT", that does seem better 😄